pyface/ui/wx/image_widget.py

```diff
@@ -96,7 +109,7 @@ def __init__(self, parent, **traits):
             wx.SystemSettings.GetColour(wx.SYS_COLOUR_3DHIGHLIGHT), 1, wx.SOLID
         )
 
-        return
+        return self.control
```

Comment:
nit-picky comment - in an earlier case, we didn't `self.control` in the `_create_control` method and instead chose to simply return it instead - whereas in this method, we're setting and returning `self.control`. Can i ask if there is a specific reason?

Reply:
There were some widgets with methods called in `_create_control` that expect `self.control` to not be populated.  These widgets need their create methods to be untangled and have some code moved to `_add_event_listeners` and other code moved to post-creation initialization.

For example, in this class, all of the binding methods should really be in `_add_event_listenrs`.

Since this PR is all about deprecating auto-creation, I wanted to make the smallest changes to achieve the required results.  I'll add an issue about untangling creation and initialization.

Reply:
Issue created: #995